### PR TITLE
Pre-launch items

### LINKED ISSUE #154: Update spellcheck workflow for launch
Currently the spellcheck action runs on PRs, which is useful for developing docs.
But at launch, we want to update the action to just use `cron` and manual triggers and to open an issue:

- Remove the PR trigger:
https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/e805a19264385657174c551724cf2c9fdf5942aa/.github/workflows/spellcheck.yml#L3-L5

- Uncomment these lines:
https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/e805a19264385657174c551724cf2c9fdf5942aa/.github/workflows/spellcheck.yml#L62-L81

### LINKED ISSUE #228: Add an analyses/README file
We should consider an `analyses/README.md` file to keep track of our modules.

We might adapt _some_ of the aspects of this openpbta equivalent: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/README.md

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #154 
Closes #228
Towards #278 

This PR updates some repo innards for launch:
- don't run spell check on PRs
- get the public-facing PR template ready for launch
  - After this PR is merged, we should make sure the template looks correct, so I don't want #278 to close until we check that
- add an `analyses/README.md` file

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -64,9 +58,12 @@ _Results should be [uploaded to your bucket](STUB_LINK docs on uploading results
 _In this section, tell reviewers what kind of feedback you are looking for._
 _This information will help guide their review._
 
-#### What are the computational requirements to be able to run the code in this PR?
+#### What are the software and computational requirements to be able to run the code in this PR?
```
Comment (allyhawkins) [inline .github/PULL_REQUEST_TEMPLATE.md:61]:
```suggestion
#### What are the software and computational requirements needed to be able to run the code in this PR?
```

**Review by allyhawkins — APPROVED** [review]
LGTM 🚀 
Just one wording change.

**Comment by sjspielman** [comment]
My guess (?) is spell check is hanging here indefinitely b/c I removed the PR trigger but it may still be a required to pass for merging PRs. Can someone with access to settings have a look and see if that's happening? @jaclyn-taroni @jashapiro

**Comment by jaclyn-taroni** [comment]
Better to just merge with admin privileges than investigate IMO.

**Comment by jashapiro** [comment]
> My guess (?) is spell check is hanging here indefinitely b/c I removed the PR trigger but it may still be a required to pass for merging PRs. Can someone with access to settings have a look and see if that's happening? @jaclyn-taroni @jashapiro

Switched the branch protection rule to pre-commit

**Pull request merged**